templates/admin/admin-classes.html

```diff
@@ -25,8 +28,11 @@ <h4 class="text-center">Total amount of shown classes: {{ classes|length }}</h4>
             <td class="classname_cell">{{class.name}}</td>
             <td class="teacher_cell">{{class.teacher}}</td>
             <td class="students_cell">{{class.students}}</td>
-            <td class="last_used_cell">{{class.last_used}}</td>
-            <td id="statistics_cell"><a href="/stats/class/{{class.id}}" target="_blank">Click here</a></td>
+            <td class="runs_week_cell">{{class.week_stats.runs}}</td>
+            <td class="error_percentage_week_cell">{% if class.week_stats.runs > 0 %}{{(class.week_stats.fails / class.week_stats.runs * 100)|round(2)}}{% else %}0.00{% endif %}%</td>
+            <td class="runs_cell">{{class.total_stats.runs}}</td>
+            <td class="error_percentage_cell">{% if class.total_stats.runs > 0 %}{{(class.total_stats.fails / class.total_stats.runs * 100)|round(2)}}{% else %}0.00{% endif %}%</td>
+            <td id="statistics_cell"><a href="/week_stats/class/{{class.id}}" target="_blank">Click here</a></td>
```

Comment:
I get a 404 when I try to open this link. Perhaps it should be `/class/{{class.id}}` without the prefix `/week_stats`?

Reply:
Fixed!